package.json

```diff
@@ -32,8 +32,9 @@
     "moment": "^2.24.0",
     "moment-timezone": "^0.5.27",
     "node-telegram-bot-api": "^0.40.0",
-    "web3": "^1.2.4",
-    "web3-core": "^1.2.2"
+    "rxjs": "7.0.0-beta.0",
```

Comment:
I knewwww it, I was 99% sure you was going to include this

This is also going to become handy soon I think, if we evolve Telegram (too be able to create tematic channels, with smarter filters)